.github/ISSUE_TEMPLATE/70_org_editor-checklist.md

```diff
@@ -0,0 +1,34 @@
+---
+name: "[Organizer -> Editor] Initial Editor Checklist"
+about: Checklist for Associate Editor to fill out prior to review
+title: "[PRE-REVIEW] Initial Editor Checklist"
+labels: pre-review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+### Instructions for Editor
+- If there is anything in the list below that you are unsure of, you can reply to this issue (after you created it) and tag the responsible organizers. Organizers' Github IDs are listed on [People page](https://www.journalovi.org/people.html).
+- Close this issue when all items are complete.
```

Comment:
Perhaps add the following notes (from Google Doc)

> Note to all editors: Some people prefer to be aware of any data that are tracked. Therefore, we want to remind you that most actions performed on Github are publicly recorded with timestamps (e.g., creating a repository, creating an issue, replying to an issue).

Reply:
added